ext/standard/tests/general_functions/proc_open_array.phpt

```diff
@@ -3,7 +3,8 @@ Using proc_open() with a command array (no shell)
 --FILE--
 <?php
 
-$php = getenv('TEST_PHP_EXECUTABLE');
+$php = getenv('TEST_PHP_EXECUTABLE') ?: PHP_BINARY;
```

Comment:
I'm not sure that falling back to the binary running the tests is a good idea.

Reply:
Ok, so you suggest we die() if not on the environment ?

Reply:
Hmm, are there actual (non pathological) cases where that env var isn't set?

Reply:
I got it to trigger in my environment .. $php was empty, then scriot got to this point:

`proc_open([$php, '-r', 'echo "Hello World!\n";'], $ds, $pipes);`

triggering me to post patch above, that disallows empty first element. :-)

Reply:
It seems to me that run-tests.php is supposed to set `TEST_PHP_EXECUTABLE` to the value of `PHP_BINARY` if not already set. Did you run the .php file directly? I think it's okay to fallback to `PHP_BINARY`, but other tests are likely affected as well. Or maybe we should just ignore that, and require users to set the environment variable when running the .php script directly.

Reply:
There are various tests with either the same code I used here or equivalent..so I guess this is an usually found problem...I think we should just ignore it for now.